apps/rule-manager/client/src/ts/components/helpers/getRule.ts

```diff
@@ -0,0 +1,6 @@
+export const getRule = async (ruleId: number) => {
+    const getRuleResponse = fetch(`${location}rules/${ruleId}`, {
+         method: 'GET',
+     });
+     return getRuleResponse
```

Comment:
Non-blocking: worth taking care of the status checking here? This fn's quite thin and reusing it across components would require the same checks each time – wonder if the same code could take care of get, create and update.

[Common types](https://github.com/guardian/editorial-tools-user-telemetry-service/blob/467caf5a96b9ef8a72f6f3cbf408df13a4d081d0/projects/event-lambdas/src/event-s3-lambda/index.ts#L25) for this can be useful, although `Either` is somewhat wishful thinking in that example, it's not a good match for the Scala equivalent 😀

Reply:
I've added a helper function to transform the responses from our API calls into our own type representing error response versus a success response.